c++/src/kj/hash.h

```diff
@@ -128,6 +130,13 @@ static KJ_CONSTEXPR(const) HashCoder HASHCODER = HashCoder();
 inline uint hashCode(uint value) { return value; }
 template <typename T>
 inline uint hashCode(T&& value) { return hashCode(_::HASHCODER * kj::fwd<T>(value)); }
+template <typename T, size_t N>
+inline uint hashCode(T (&arr)[N]) {
```

Comment:
This had to be here instead of as a member function on `HashCoder` because otherwise the `T[N]` was getting decayed by kj::fwd to `T*`.